app/src/views/Game.vue

```diff
@@ -7,13 +7,14 @@
 
   <template v-if="gameState">
     <Opponents :players="opponents" />
-    <Deck
-      :is-spectator="isSpectator"
-      :is-user-turn="isUserTurn"
-      :cards-in-deck="cardsInDeck"
-      :current-player="currentPlayer"
-      @draw="drawCard"
-    />
+    <div class="GameBoard">
+      <button :disabled="!isUserTurn" @click="drawCard">
+        <Card :text="cardsInDeck" />
+      </button>
```

Comment:
No need to wrap the `Card` component in a button
```suggestion
      <Card tag="button" :text="cardsInDeck" :disabled="!isUserTurn" @click="drawCard" />
```